fastembed/rerank/cross_encoder/onnx_text_model.py

```diff
@@ -100,7 +98,7 @@ def _rerank_pairs(
         is_small = False
 
         if isinstance(pairs, tuple):
-            pairs = [pairs]
+            pairs = list(pairs)
```

Comment:
the intended behaviour is 

```
("Query_1", "Document_1") -> [("Query_1", "Document_1")]
```

not 

```
("Query_1", "Document_1") -> ["Query_1", "Document_1"]
```

though, I understand that ("Query_1", "Document_1") is a violation of the type hint `Iterable[tuple[str, str]]`, but I would rather not break this behaviour